BedrockServer.cpp

```diff
@@ -836,6 +838,18 @@ void BedrockServer::worker(SData& args,
             canWriteParallel = canWriteParallel && (state == SQLiteNode::MASTERING);
             canWriteParallel = canWriteParallel && (command.writeConsistency == SQLiteNode::ASYNC);
 
+            // If all the other checks have passed, and we haven't sent a quorum command to the sync thread in a while,
+            // auto-promote one.
+            if (canWriteParallel) {
+                uint64_t now = STimeNow();
```

Comment:
this is only used once, seems like you could just call STimeNow() in the line below

Reply:
It's used twice.

Reply:
oh shit, missed that 👍